boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -186,6 +194,30 @@ class FsmStateMachineTest : public testing::Test
         }
     }
 
+    void CheckBinarySwitchCanSignalsInAllStates(
+        bool &fake_value,
+        uint8_t (*can_signal_getter)(const struct FsmCanTxInterface *),
+        uint8_t on_choice,
+        uint8_t off_choice)
+    {
+        for (const auto &state : GetAllStates())
+        {
+            SetInitialState(state);
+
+            fake_value = on_choice;
+            App_SharedStateMachine_Tick100Hz(state_machine);
+            ASSERT_EQ(
+                on_choice,
+                App_CanTx_GetPeriodicSignal_BRAKE_STATUS(can_tx_interface));
+
+            fake_value = off_choice;
```

Comment:
Continuing with what I mentioned here, the fake value should really be the boolean value for `is_something_on`. We don't have to and shouldn't force the `fake_value` to match the `on/off_choice`.
```suggestion
            fake_is_turned_on = true;
            App_SharedStateMachine_Tick100Hz(state_machine);
            ASSERT_EQ(
                on_choice,
                App_CanTx_GetPeriodicSignal_BRAKE_STATUS(can_tx_interface));

            fake_is_turned_on = false;
```

Reply:
fixed